src/index.html

```diff
@@ -13,6 +13,136 @@
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
+
+    <div class="stars stars--3">
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
+
+    <div class="stars stars--4">
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
+
+    <div class="stars stars--5">
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars__sizing">
+        <div class="stars__star"></div>
+      </div>
+    </div>
```

Comment:
Add empty lines between sibling blocks of HTML to improve readability. For example, add an empty line between the closing </div> of one .stars block and the opening <div> of the next .stars block.